### PR TITLE
Improve GitHub & Jenkins installation flow

### PR DESCRIPTION
**Pull request by sverdlov93**
- [ ] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] This pull request is on the dev branch.
- [ ] I used gofmt for formatting the code before submitting the pull request.
---

github - https://github.com/sverdlov93/frogbot/blob/Improve-docs/docs/install-github.md
jenkins - https://github.com/sverdlov93/frogbot/blob/Improve-docs/docs/templates/jenkins/README.md

### CODE REVIEW

File: commands/utils/params.go [hunk 1]
```diff
@@ -34,18 +32,23 @@ var (
...
 	ServerDetails *coreconfig.ServerDetails
-	Client        vcsclient.VcsClient
+	GitClient     vcsclient.VcsClient
 	ReleasesRepo  string
 }
 
 type RepoAggregator []Repository
 
+// NewRepoAggregator returns an initialized RepoAggregator with an empty repository
+func NewRepoAggregator() RepoAggregator {
```
Comment (eyalbe4) [inline commands/utils/params.go:45]: Looks like this function can be made private.

File: commands/utils/params_test.go [hunk 2]
```diff
@@ -23,15 +23,17 @@ func TestExtractParamsFromEnvError(t *testing.T) {
...
 
 	SetEnvAndAssert(t, map[string]string{JFrogUrlEnv: "http://127.0.0.1:8081"})
-	_, err = extractJFrogCredentialsFromEnv()
+	_, err = extractJFrogCredentialsFromEnvs()
 	assert.EqualError(t, err, "JF_USER and JF_PASSWORD or JF_ACCESS_TOKEN environment variables are missing")
 }
 
-func TestExtractParamsFromEnvPlatform(t *testing.T) {
+// Test extraction of env params in ScanPullRequest command
+// Pull request ID is not default, which mean we don't have branches related variables defined.
```
Comment (eyalbe4) [inline commands/utils/params_test.go:35]: Wring grammar in this line.

File: commands/utils/params_test.go [hunk 3]
```diff
@@ -40,12 +42,29 @@ func TestExtractParamsFromEnvPlatform(t *testing.T) {
...
 		GitRepoEnv:          "frogbot",
 		GitTokenEnv:         "123456789",
-		GitBaseBranchEnv:    "dev",
 		GitPullRequestIDEnv: "1",
 	})
 	extractAndAssertParamsFromEnv(t, true, true)
 }
 
+// Test extraction in ScanRepository command
+// Pull request ID is default 0, which mean we will have branches related variables.
```
Comment (eyalbe4) [inline commands/utils/params_test.go:51]: Wring grammar in this line.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
The `create-fix-pull-requests` phrase was left unreplaced in `install-github.md`.

**Review by EyalDelarea — APPROVED** [review]
Reviewed the Jenkins Docs, looks amazing!

**Pull request merged**